src/sidebar/store/modules/direct-linked.ts

```diff
@@ -121,9 +107,6 @@ function setDirectLinkedGroupFetchFailed() {
   });
 }
 
-/**
- * Clear the direct linked group fetch failure.
- */
```

Comment:
This comment and the ones above felt unnecessary, as they are basically the same as the function name.